### DISCUSSION TITLE
Does messageBuilder work in Watchface or not?

### DISCUSSION THREAD

**Discussion by spyx1** [zepp-health/zeppos-forum#293] [Q&A]
Hello. 
I became the proud owner of a Bip 5 watch. I’m trying to create my own watchface and there was a need to load data via fetch api. Everything works in the emulator, but when I install my watchface on the watch, the data is not loaded.
I need someone to confirm that messageBuilder works not only on pages and widgets, but also on watchface.

**Comment by MyoungXUE** [comment]
Hello, the Watchface does not support `MessageBuilder`. Only Mini Program support `MessageBuilder`.
